docs/Installation.md

```diff
@@ -359,6 +363,10 @@ sudo install -v -m 755 ./zm-testagent.lsb /etc/init.d/zm-testagent
 sudo install -v -m 755 ./tmpfiles.conf /usr/lib/tmpfiles.d/zonemaster.conf
 ```
 
+> If this is an update of Zonemster-Backend, you should remove any script from older
+> version of Zonemaster-Backend in `/etc/init.d`, i.e. any for Zonemaster except for
+> the ones just installed.

```

Comment:
Maybe refer to /etc/init.d/zm-backend.sh instead?

Reply:
Good suggestion. Will fix.